### PR TITLE
Creates the Cockroach Target Documentation

### PR DESCRIPTION
**Pull request by JeffNeff**

### CODE REVIEW

**Review by cab105 — APPROVED** [review]
Very minor nit.

File: docs/targets/cockroach.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+This event target receives [CloudEvents][ce] over HTTP and sends them to [CockroachDB](https://www.cockroachlabs.com/product/)
+
+## Prerequisite(s)
+
+- A CockroachDB cloud deployment.
+- A [connection string](https://www.cockroachlabs.com/docs/v21.1/connection-parameters) for the deployment.
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a target of type `CockroachDB`.
```
Comment (sameersbn) [inline docs/targets/cockroach.md:12]: nit: Do we really need to capitalize Bridge?

File: docs/targets/cockroach.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Arbitrary
+This target will consume arbitrary events and upload them into the pre-configured default table with the Cloudevent ID as the Primary key. 
+
+### io.triggermesh.cockroach.query
+
+Events of this type contain nuanced data that is used to query a CockroachDB.
+
+This type expects a [JSON][ce-jsonformat] payload with the following properties:
+
+| Name  |  Type |  Comment | Example
```
Comment (sameersbn) [inline docs/targets/cockroach.md:36]:
```suggestion
| Name  |  Type |  Comment | Example |
```

File: docs/targets/index.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ The following is a list of Triggermesh event destinations known as `Targets`. So
     - [AWS SQS](./awssqs.md): Forward event payload to AWS Simple Queueing Service (SQS).
     - [AWS Kinesis](./awskinesis.md): Forward event payload to AWS Kinesis.
     - [AWS EventBridge](./awseventbridge.md): Forward arbitrary events to AWS EventBridge.
+- [Cockroach](./cockroach.md): Store events or create custom queries.
```
Comment (sameersbn) [inline docs/targets/index.md:14]: Should it be CockroachDB?

**Pull request closed without merging**